### PR TITLE
PDP-684: Path verification in hugging face repository train endpoint

### PR DESCRIPTION
**Pull request by AdrianaMCarmiol**
Encode test was failing so I changed it to have a small error margin of comparison between vectors.

### CODE REVIEW

File: src/test/model_test.py [hunk 1]
```diff
@@ -450,7 +454,10 @@ def test_model_encode(self):
 
         model_class = ModelSentenceTransformer(MODEL_ROOT)
         response_answer = model_class.encode("123", DATA_TEST_SINGLE_TEXT, 'deepset/roberta-base-squad2')
-        self.assertEqual(str(response_answer), answer)
+        answer = json.loads(answer)
+        result = np.abs(np.subtract(answer['result'], response_answer['result']))
+        average = sum(sum(result)) / (result.shape[0] * result.shape[1])
+        self.assertTrue(threshold > average)
```
Comment (ricardo-leon-pureinsights) [inline src/test/model_test.py:460]: I like this approach @AdrianaMCarmiol. What's the average value typically? Maybe the threshold is currently to high (0.1).

Comment (AdrianaMCarmiol) [reply src/test/model_test.py:460]: Normally the difference between the vectors ends up being around 1x10<sup>-7</sup>

Comment (ricardo-leon-pureinsights) [reply src/test/model_test.py:460]: In that case I think you can make the threshold more strict, something like 0.001.

**Review by ricardo-leon-pureinsights — COMMENTED** [review]
Looks good to me!

**Review by andresmarenco-pureinsights — APPROVED** [review]

**Pull request merged**